src/base/bittorrent/torrentinfo.cpp

```diff
@@ -28,6 +28,7 @@
 
 #include "torrentinfo.h"
 
+#include <boost/optional.hpp>
```

Comment:
are you sure this is needed? I didn't spot a usage of it in the diff.

Reply:
At first I was going to do it all (libtorrent-1.2 support) at once, but then I decided to divide this task into several stages. So I narrowed these changes down. Maybe it's from there. Then it will need further. Or I'll delete it otherwise.

Reply:
OK. This isn't worth a new commit on its own anyway.